test/contract-factory.test.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from "chai";
+import { starknet } from "hardhat";
+import { TIMEOUT } from "./constants";
+import { StarknetContractFactory } from "hardhat/types/runtime";
+import { getPredeployedOZAccount } from "./util";
+import { OpenZeppelinAccount } from "@shardlabs/starknet-hardhat-plugin/dist/src/account";
+import * as fs from "fs";
+import { StarknetContract } from "@shardlabs/starknet-hardhat-plugin/dist/src/types";
+
+describe("StarknetContractFactory", function () {
+    this.timeout(TIMEOUT);
+    let contractFactory: StarknetContractFactory;
+    let account: OpenZeppelinAccount;
+    // let contract: StarknetContract;
```

Comment:
What about this?

Reply:
Cleared ;)

Reply:
I had contract and contractFactory tests in the same file in the beginning, but then the file seemed to be getting large so refactored it.